.gitignore

```diff
@@ -3,3 +3,4 @@ artifacts/
 npm-debug.log
 .DS_STORE
 .*.swp
+package-lock.json
```

Comment:
I think Petey suggested before that we commit this file. 
We shouldn't ignore it?